lib/fizzy/execute.cpp

```diff
@@ -449,6 +449,36 @@ inline void comparison_op(OperandStack& stack, Op<T> op) noexcept
     stack.top() = uint32_t{op(val1, val2)};
 }
 
+template <typename T>
+inline constexpr T add(T a, T b) noexcept
+{
+    return a + b;
```

Comment:
I'd prefer we add the asserts for is_integral||is_float to avoid any random overloading.

Reply:
This is big overkill and still is not complete (as `bool`, `char`, `long double` would be a match). I can't imagine accidentally adding two things.

Reply:
Realistically what could happen is that Value is overloaded and this matches for Value+Value. Why would it be an overkill?

Reply:
This function is perfectly valid for any type that is copyable and has + operator. The `Value` does not support + operator as of #464. Also all usages of it explicitly specify the type.